commit 5d67c4e3a3f5314f45e6c09cbda6a76d451c9d48
Author: Tim Gover <tim.gover@raspberrypi.org>
Date:   Wed Sep 5 22:09:28 2018 +0100

    rpi-simple-soundcard: Use nicer driver name "RPi-simple"
    
    Rename the driver from "RPI simple soundcard" to "RPi-simple" so that
    the driver name won't be mangled allowing to be used unaltered as the
    card conf filename.

diff --git a/sound/soc/bcm/rpi-simple-soundcard.c b/sound/soc/bcm/rpi-simple-soundcard.c
index 811345d6a61d..7fa92e29502a 100644
--- a/sound/soc/bcm/rpi-simple-soundcard.c
+++ b/sound/soc/bcm/rpi-simple-soundcard.c
@@ -202,7 +202,7 @@ static const struct of_device_id snd_rpi_simple_of_match[] = {
 };
 
 static struct snd_soc_card snd_rpi_simple = {
-	.driver_name  = "RPI simple soundcard",
+	.driver_name  = "RPi-simple",
 	.owner        = THIS_MODULE,
 	.dai_link     = NULL,
 	.num_links    = 1, /* Only a single DAI supported at the moment */
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1202_5d67c4e3a3f5314f45e6c09cbda6a76d451c9d48.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1202_5d67c4e3a3f5314f45e6c09cbda6a76d451c9d48.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
